src/icatcher/options.py

```diff
@@ -54,6 +54,17 @@ def parse_arguments(my_string=None):
                         help="the type of dataset to preprocess")
     parser.add_argument("--illegal_transitions_path", type=str, help="path to CSV with illegal transitions to 'smooth' over")
     parser.add_argument('--version', action='version', version="%(prog)s "+version)
+    # face detection options:
+    parser.add_argument("--num_cpus_saved", type=int, default=0,
+                        help="amount of cpus to not use in parallel processing of face detection")
+    parser.add_argument("--fd_batch_size", type=int, default=16,
+                        help="amount of frames fed into face detector at one time for batch inference")
+    parser.add_argument("--fd_confidence_threshold", type=float, default=0.9,
```

Comment:
The previous default (0.7) for cvdnn is not preserved. Is there some elegant way to make this 0.7 if user selects cvdnn?